test/gui/tst_removeAccount/test.feature

```diff
@@ -0,0 +1,25 @@
+Feature: remove account
+
+  As a user
+  I want to remove my account
+  So that I won't be using any client-UI services
+
+
+  Scenario: remove an account
+    Given user "Alice" has been created on the server with default attributes and without skeleton files
+    And user "Brian" has been created on the server with default attributes and without skeleton files
+    And user "Alice" has set up a client with default settings
+    And the user has added an account with
+        | server      | %local_server%     |
+        | user        | Brian              |
+        | password    | AaBb2Cc3Dd4        |
+        | localfolder | %client_sync_path% |
+    When the user removes the account
```

Comment:
which account is to be removed? You have set up two

Reply:
This will remove the account that was added later on. In this case the user that will be removed is `Brian`

Reply:
That need to be specified in the step (for less astonishment ;-) )